src/corehost/cli/runtime_config.cpp

```diff
@@ -7,19 +7,23 @@
 #include "utils.h"
 #include "cpprest/json.h"
 #include "runtime_config.h"
+#include "roll_fwd_on_no_candidate_fx_option.h"
 #include <cassert>
 
-
 // The semantics of applying the runtimeconfig.json values follows, in the following steps from
 // first to last, where last always wins. These steps are also annotated in the code here.
-// 1) Apply the environment settings
+// 0) Start with the default values
+// 1) Apply the environment settings for DOTNET_ROLL_FORWARD_ON_NO_CANDIDATE_FX
 // 2) Apply the values in the current "runtimeOptions" section
 // 3) Apply the values in the referenced "frameworks" section
-// 4) Apply the overrides (from command line or other)
+// 4) Apply the environment settings for DOTNET_ROLL_FORWARD
+// 5) Apply the overrides (from command line or other)
 
 runtime_config_t::runtime_config_t()
```

Comment:
If we passed an interface here we could possibly unit test this better.  
```C++
class IRuntimeConfigSource
{
 virtual pal::string_t getDOTNET_ROLL_FORWARD_ON_NO_CANDIDATE_FX() = 0;
...
};
```

Reply:
I agree that eventually having interfaces to mock will help. In this case it would help only a little. If it's OK, I won't make that change here yet.